### DISCUSSION TITLE
Help Menu & User Flow

### DISCUSSION THREAD

**Discussion by karenychen** [Azure/draft#33] [General]
moving the actual content to wiki since Github doesn't support font colors.

[Link to UX design doc](https://msazure.visualstudio.com/CloudNativeCompute/_wiki/wikis/CloudNativeCompute.wiki/234397/Draftv2-UX-Design)

**Comment by imiller31** [comment]
I think this all looks great! The spinner idea is nice. I think adding some colors to the experience would also be nice.

**Reply by qpetraroia** [reply]
Some emojis too!
